### ISSUE TITLE
Improve article 2201.12093

### ISSUE THREAD

**Issue by littlewwwhite** [dginev/ar5iv#430]
**Exact location of issue**

Article fails to convert.

**Problem details**

Error cascade, starting from a pgf math parsing failure in latexml [log](https://ar5iv.labs.arxiv.org/log/2201.12093):

```
 (Processing content /dev/shm/Rc_mmCWJel/images/ablation_K.tex...

Error:latex:(PGF Math) Package PGF Math Error: 
Sorry, an internal routine of the floating point unit got an ill-formatted floating point number `0.19'. 
The unreadable part was near '0.19'. (in '-0.19') See the PGF Math package documentation for explanation.
```

**Comment by dginev** [comment]
Thank you for the report! I have added more details to the description.

**Comment by dginev** [comment]
With the pgf fixes that landed in latexml v0.8.8, this article now has healthy HTML:

https://ar5iv.labs.arxiv.org/html/2201.12093

I'll resolve here.
